packages/react/package.json

```diff
@@ -26,7 +26,7 @@
   ],
   "scripts": {
     "prebuild": "rimraf dist",
-    "build": "yarn run build:icons && yarn run build:ts && yarn run build:catalog",
+    "build": "yarn run build:ts && yarn run build:catalog",
```

Comment:
When will icons be built as part of CI or local development on a clean repo?

Reply:
Right now it is generating files that are checked into git so we only need to run it when we need the icons to change. A dev won't need to run this during local development. If we wanted, we could have it directly generate finalized build artifacts instead, then we would add this back.

Reply:
Ooh! Good call.  Glad to have this removed then 👍